packages/storybook8/stories/Concepts/AdhocCalling/Docs.mdx

```diff
@@ -0,0 +1,65 @@
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+
+import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippet.tsx';
+import AdhocCallingVoiceAppSnippetText from '!!raw-loader!./snippets/AdhocCallingVoiceApp.snippet.tsx';
+
+<Meta title="Concepts/Ad-hoc calling" />
+
+# Ad-hoc calling
+
+## Overview
+
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or multiple teams users. You cannot call multiple
+Teams voice applications at the same time.
+
+## Calling a Teams voice application
+
+To call a Teams voice application like a call queue or auto attendant, you need to use the id of the Teams resource account. Check out this [tutorial](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-auto-attendant#find-object-id-for-auto-attendant) on how to find
+this id. The id of the Teams resource account should be stored in the type of `MicrosoftTeamsAppIdentifier[]` and given to the `targetCallees` property
+of the [CallAdapter](?path=/docs/composite-adapters--page#call-adapter) when creating a call. Now when the [CallComposite](?path=/docs/composites-call-basicexample--basic-example) starts up it will call the Teams voice application
+from the configuration screen. See the following snippet for an example.
+
+<MessageBar messageBarType={MessageBarType.warning}>
+  Note: Teams resource account ids linked to a call queue need to be in format '28:orgid:UUID' when used in the
+  CallParticipantsLocator. For example, '28:orgid:87d349ed-44d7-43e1-9a83-5f2406dee5bd'.
+</MessageBar>
+
+<Source code={AdhocCallingVoiceAppSnippetText} />
+
+The two callback functions `onFetchProfile` and `onFetchAvatarPersonaData` provided at the bottom of the snippet are
+for customizing display names and avatars of users, respectively.
+
+## Calling a Teams user
+
+Like a Teams voice application, you can call a Teams user by using their id. The id of the Teams user should be stored in the type of `MicrosoftTeamsUserIdentifier[]`.
+This array should be given to the `targetCallees` property of the [CallAdapter](?path=/docs/composite-adapters--page#call-adapter) when creating a call. Like with the bot, the
```

Comment:
```suggestion
This array should be given to the `targetCallees` property of the [CallAdapter](?path=/docs/composites-adapters--docs#call-adapter) when creating a call. Like with the bot, the
```

Reply:
Committed your link fix changes. Thanks for providing them!